bokeh/io.py

```diff
@@ -346,6 +356,47 @@ def _save_helper(obj, filename, resources, title, validate):
         with io.open(filename, "w", encoding="utf-8") as f:
             f.write(decode_utf8(html))
 
+def savepng(obj, filename=None, state=None, validate=True):
```

Comment:
should it be `save_png` with an underscore?

Why did you go with a separate `savepng` vs. making `save()` work on png output?